src/wireviz/wv_colors.py

```diff
@@ -112,7 +112,7 @@ def get_color_hex(input, pad=False):
     try:
         output = [_color_hex[input[i:i + 2]] for i in range(0, len(input), 2)]
     except KeyError:
-        print("Unknown color specified")
+        print(f"Unknown color specified: {input}")
```

Comment:
I tried that, but I'm not sure if I like the byte indicator around the color string. `Unknown color specified: b'OGYW'` But I will include it if you want :smile:

Reply:
It's easy to avoid the byte indicator, and also the quotes when not needed:
```py
quoted = str(bytes(input, "utf-8"))[1:] # Skip byte indicator, but keep quotes
print(f'Unknown color specified: {input if input.strip() == quoted[1:-1] else quoted}')
```
However, I don't insist. It's just a suggestion in case someone find it useful. 😄

Reply:
Maybe I'm out of the loop, but why don't we stick to the suggested
```python
print(f"Unknown color specified: {input}")
```
?
Are we really worried people will try to use emojis and non-printable characters as colors? 😛

Reply:
Any of them are OK with me. I just wanted to menton ideas that have been useful in other projects, but you decide what is needed in this project. My reply was not intended as a fight for my suggestion, but simply a correction of the draw-back that was commented.

Reply:
OK, just trying to understand the motivation for the idea. I think for now it's safe to keep it as-is :)